### ISSUE TITLE
iPad: Reports pie chart loading

### ISSUE THREAD

**Issue by dshr** [toggl/mobileapp#4922]
## 🐛 Bug Report
Sometimes the pie chart doesn't load by default, only after you touch the view.

## Context
Build/Branch: project/ipad
Phone OS: 12.1
Phone model: iPad Mini

## Expected behavior
It should load right away.

## To reproduce
Open reports view several times.

**Comment by daividssilverio** [comment]
This seems to be working fine on the latest test build. Tested on my iPad mini.

**Comment by sofokli1** [comment]
This could happen in bad network connection

**Comment by sofokli1** [comment]
Unless we can reproduce this, I say we close this too. Also since we did not do any special change to the VM for this, this would be an issue for iPhones and maybe android too.

**Comment by amulware** [comment]
I'll assuming this probably is no longer relevant with the rewrite.
